### ISSUE TITLE
Misplacement of Albacete region in Spain

### ISSUE THREAD

**Issue by jata84** [whosonfirst-data/whosonfirst-data-admin-es#40]
I am opening this issue to address a discrepancy found in a pull request I submitted via the web form. https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/39

The pull request involves changing the region from Albacete to Álava. This change isn't merely a translation alteration; rather, it corrects the geographical representation. Albacete is situated in the southern region of Spain, as depicted [here](https://www.google.com/maps/place/Albacete/@39.9945895,0.7358732,7.25z/data=!4m6!3m5!1s0xd667c1d58c47ad1:0x30340f3be4bc950!8m2!3d38.8036957!4d-1.9881527!16zL20vMDEyNDhs?entry=ttu).

Conversely, Álava is located in the northern part of Spain, as shown [here](https://www.google.com/maps/place/%C3%81lava/@42.6107525,-1.6193495,9.75z/data=!4m6!3m5!1s0xd4fc01662943233:0x304f58273c9c940!8m2!3d42.9099989!4d-2.6983868!16zL20vMGh2azc?entry=ttu).

Upon review, it's apparent that Albacete is incorrectly placed in two different locations in your database. The entry in the northern part of Spain should indeed be listed as Álava.

For further reference, you can view and edit the data directly through this link: [Edit Link](https://writefield.nextzen.org/place/edit?url=https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-es/master/data/856/826/15/85682615.geojson).

In summary, the issue highlights the misplacement of Albacete in two separate locations, with the correct designation being Álava for the northern region of Spain. Additional information and verification can be found [here](https://spelunker.whosonfirst.org/search/?q=albacete).

**Comment by stepps00** [comment]
Closing in preference of https://github.com/whosonfirst-data/whosonfirst-data/issues/2188
